templates/adp-backend-template-node/template.yaml

```diff
@@ -198,6 +198,84 @@ spec:
         owner: ${{ parameters.repo_url | parseRepoUrl | pick('owner') }}
         permission: push
 
+    - id: configureAdoProject
+      name: Configure Azure DevOps Project
+      action: http:backstage:request
+      input:
+        method: PATCH
+        path: /proxy/adp-portal-api/AdoProject/${{ parameters.ado_project }}/onboard
+        headers:
+          content-type: 'application/json'
+        body:
+          environments:
+          - name: SND3
+            description: SND environment
+          - name: DEV1
+            description: DEV1 environment
+          - name: TST1
+            description: TST1 environment
+          - name: TST2
+            description: TST2 environment
+          - name: PRE1
+            description: PRE1 environment
+          - name: PRD1
+            description: PRD1 environment
+          serviceConnections:
+          - AZD-ADP-SND1
+          - AZD-ADP-SND2
+          - AZD-ADP-SND3
+          - AZD-ADP-SSV3
+          - AZD-ADP-DEV1
+          - AZD-ADP-TST1
+          - AZD-ADP-PRE1
+          - AZD-ADP-SNYK
+          - Defra-ADP-Github
+          - DEFRA SonarCloud
+          - DEFRA
+          agentPools:
+          - DEFRA-ADP-SND1-ubuntu2204
+          - DEFRA-ADP-SND3-ubuntu2204
+          - DEFRA-ADP-DEV1-ubuntu2204
+          - DEFRA-ADP-TST1-ubuntu2204
+          - DEFRA-ADP-PRE1-ubuntu2204
+          variableGroups:
+          - name: ${{ parameters.component_id }}-snd3
```

Comment:
this can just be <environment> now with only a single import @kbabigumira  @ppotturi  @ajaysoni-cap  ?